common-rest/src/main/java/org/uniprot/api/rest/output/converter/AbstractUUWHttpMessageConverter.java

```diff
@@ -237,7 +237,7 @@ private void logStats(int counter, Instant start) {
         long millisDuration = Duration.between(start, now).toMillis();
         int secDuration = (int) millisDuration / 1000;
         String rate = String.format("%.2f", ((double) counter) / secDuration);
-        log.debug(
+        log.info(
```

Comment:
We can keep it debug, what do you think?

Reply:
Yes, changed it to debug